### PR TITLE
add http basic page

### PR DESCRIPTION
**Pull request by dylangrandmont**
Add Integration docs for HTTP Basic Auth

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -24,6 +24,7 @@ nav:
               - Connecting Cloud Storage: advanced-usage/connecting-cloud-storage/index.md
               - S3-Compatible APIs: advanced-usage/connecting-cloud-storage/s3-compatible.md
               - GCP: advanced-usage/connecting-cloud-storage/gcp.md
```
Comment (gordonhart) [inline mkdocs.yml]: Not added in this PR, but I missed it first time around — is there a better (more complete) name than `GCP` for this integration?

Comment (dylangrandmont) [reply mkdocs.yml]: Any of `Google Cloud Platform` / `Google Cloud` / `Google Cloud Storage` could be considered complete; any preference of those three? (I'd like to leave the page file name as it's already linked from the app)

Comment (dylangrandmont) [reply mkdocs.yml]: I think my personal preference would be `Google Cloud Storage` as this is most specifically what we are connecting to

File: docs/advanced-usage/connecting-cloud-storage/http-basic.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+Integrations can be established using HTTP Basic Auth.
+
+To get started, ensure you have admin access within Kolena.
+Navigate to your [Organization Settings's Integration Tab](https://app.kolena.io/redirect/organization?tab=integrations) and click "Add Integration", then "HTTP Basic".
+
+### 1. Save Integration on Kolena
+
+On to the Kolena platform [Integration Tab](https://app.kolena.io/redirect/organization?tab=integrations), fill in the fields for the integration and then click "Save".
```
Comment (ly-kolena) [inline docs/advanced-usage/connecting-cloud-storage/http-basic.md]: `On to` is this intentional?

File: docs/advanced-usage/connecting-cloud-storage/http-basic.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+| Field | Description |
+|---|---|
+| URL Origin | The origin of the domain you wish to load data from. Ensure you omit the protocol (e.g. `https://`) |
+| Username | The service level username for your http basic auth system |
```
Comment (ly-kolena) [inline docs/advanced-usage/connecting-cloud-storage/http-basic.md]: Nit: is `service level` well understood in this context?

**Review by ly-kolena — APPROVED** [review]

File: docs/advanced-usage/connecting-cloud-storage/http-basic.md (hunk 3 above)
Comment (dylangrandmont) [reply docs/advanced-usage/connecting-cloud-storage/http-basic.md]: Good question. I think `username for your http basic auth system`  is sufficiently clear in this context so I will update that and `password`

**Pull request merged**